charts/osm-arc/values.yaml

```diff
@@ -8,8 +8,8 @@ Azure:
     Distribution: <cluster_distribution>
 
 OpenServiceMesh:
-    ignoreNamespaces: "kube-system azure-arc arc-osm-system"
-    testedDistros: "aks gke eks openshift rancher_rke"
+  ignoreNamespaces: "kube-system azure-arc arc-osm-system"
+  testedDistros: "aks gke eks openshift rancher_rke"
```

Comment:
At some point we should convert these to comma-separated values for readability / robustness.